### ISSUE TITLE
Why not CSS Preprocessor?

### ISSUE THREAD

**Issue by emzero** [barbar/vortigern#62]
Any reason not to use SCSS (or even LESS if you want)?

Sass SCSS seems to be the popular choice of CSS preprocessing in React projects.

**Comment by batuhan** [comment]
Hey @enzore,

First of all, thanks for the feedback. 

We've started with SCSS but later we've opted out of it in favor of cssnext. 

In case you didn't know, cssnext is based on the latest CSS specs and therefore it's future proof. And the feature set is really comparable. 

One more thing I should add is, Vortigern wasn't build as a technology showcase project but instead for production use at Barbar. This means before we choose a library to use we carefully examine every option and try to make sure we are not creating any technical debt. 

Thanks again,
Feel free to ask anything else,
Batuhan.

**Comment by batuhan** [comment]
Hey @enzore,

First of all, thanks for the feedback. 

We've started with SCSS but later we've opted out of it in favor of cssnext. 

In case you didn't know, cssnext is based on the latest CSS specs and therefore it's future proof. And the feature set is really comparable. 

One more thing I should add is, Vortigern wasn't build as a technology showcase project but instead for production use at Barbar. This means before we choose a library to use we carefully examine every option and try to make sure we are not creating any technical debt. 

Thanks again,
Feel free to ask anything else,
Batuhan.
